### PR TITLE
Adds Playing Card Model

### PR DESCRIPTION
**Pull request by moby-dicc**
Adds Playing Card Model. Please the check the file structure @ankitjena @aquibbaig

### CODE REVIEW

**Comment by ankitjena** [comment]
@moby-dicc Could you share some screenshots of the work

**Comment by aquibbaig** [comment]
@ankitjena He can't maybe cause it's just a class declaration.

**Comment by aquibbaig** [comment]
# Proposal for cards
Rather than having spades, clubs, hearts and diamonds, we can have custom cards like UNO.
# Card classes
- Reds
- Blues
- Greens
- Blacks
# Card Sets
There will be 2 set of card-groups.
## Set 1
A, 2, 3, 4, ... 6.
## Set 2
8, 9, 10, J, Q, K.
# Goal
The goal is to make one set from each class.

**Comment by aquibbaig** [comment]
@moby-dicc @ankitjena

File: lib/models/playing_cards.dart [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+enum CardColor {
+  red,
+  black,
+}
+
+// Simple playing card model
+class PlayingCard {
+  CardSuit cardSuit;
+  CardType cardType;
+  bool faceUp;
```
Comment (aquibbaig) [inline lib/models/playing_cards.dart:35]: We are not doing that. Every card is face down(hidden) until is sent to a "graveyard" after a set is done.

Comment (moby-dicc) [reply lib/models/playing_cards.dart:35]: I'll try again and make changes accordingly tonight

Comment (aquibbaig) [reply lib/models/playing_cards.dart:35]: What do you think about #6 ?

**Comment by moby-dicc** [comment]
@aquibbaig please review the changes

File: lib/models/playing_cards.dart [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  red,
+  black,
+}
+
+// Simple playing card model
+class PlayingCard {
+  CardSuit cardSuit;
+  CardType cardType;
+  bool opened;
+
```
Comment (aquibbaig) [inline lib/models/playing_cards.dart:35]: A card also needs to have a field -> "name" which shows the name of the player to which it belongs to (currently) and it is a required field.

File: lib/models/playing_cards.dart [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+enum CardColor {
+  red,
+  black,
+}
+
+// Simple playing card model
+class PlayingCard {
+  CardSuit cardSuit;
+  CardType cardType;
+  bool opened;
```
Comment (aquibbaig) [inline lib/models/playing_cards.dart:34]: Why do we need this "opened" field here? Is there some point am I missing here?

**Pull request merged**